src/draw.cpp

```diff
@@ -710,11 +710,11 @@ void KDraw::draw_porttextbox(eBubbleStyle bstyle, int chr)
     a--;
     linexofs = a * 12;
 
-    menubox(double_buffer, 19, 172 - linexofs, 4, 4, BLUE);
-    menubox(double_buffer, 66, 196 - linexofs, party[chr].name.length(), 1, BLUE);
+    menubox(double_buffer, 19, 154 - linexofs, 4, 4, BLUE);
```

Comment:
Do we have any idea what these magic numbers are? `172` isn't evenly divisible by 16 (TILE_W), nor is `154` (which is now 18 less than 172).

If they're offsets from the left edge of, say, a border/frame, then a comment or `const int FrameXoffset = ...;` would be useful to see what it implies.

If it's just "it looks good to my eyeball", that's okay, but I'd like to remove as many "magic numbers" that I find in the code as I can.

Reply:
Yes its just what looks good. This function needs some cleanup as why is a line X offset being subtracted from the Y coordinate?